src/_includes/mac-path.md

```diff
@@ -0,0 +1,5 @@
+{{site.alert.important}}
+  Ensure that the **Homebrew `bin` directory is in your `PATH`** if you'd like
```

Comment:
Hmmm... our style is to have the IF part first, but that seems to bury the important part. How about something like this:

**Make sure the Homebrew...PATH**. Setting up the path correctly makes it easier to use Dart SDK commands such as `dart` and `dartfmt`.

Reply:
How about this:
>   Make sure the **Homebrew `bin` directory is in your `PATH`**. Setting up the path correctly makes it easier to use Dart SDK commands such as `dart` and `dartfmt`. For help setting up your path, consult the [Homebrew FAQ.](https://docs.brew.sh/FAQ)